src/content_scripts/protoots.js

```diff
@@ -105,39 +105,34 @@ function main() {
  */
 async function fetchPronouns(statusID, account_name) {
 	// log(`searching for ${account_name}`);
-
-	let cacheResult = await browser.storage.local.get().then(getSuccess, onError);
+	let cacheResult = { pronounsCache: {} };
+	try {
+		cacheResult = await browser.storage.local.get();
```

Comment:
could we use something like `{cacheResult || { pronounsCache: {} }}`?
i've seen that method used in some places, but am not sure whether that works in this case. would have to test.

Reply:
so, you mean that the fallback is only defined if the await to the storage fails? 
Theoretically it should work, yes.